stimela/backends/flavours/python_flavours.py

```diff
@@ -154,13 +154,21 @@ def get_arguments(self, cab: Cab, params: Dict[str, Any], subst: Dict[str, Any],
             msg4 = f"""print("## return value is ", _result)"""
         else:
             msg1 = msg2 = msg3 = msg4 = ""
+
+        pre_command_str = ""
+        if self.pre_commands:
+            pre_command_str += "".join(self.pre_commands.values())
```

Comment:
Should probably join with "\n", since single-line bits of YAML will not have a trailing newline in the value.